pkg/cache/cache_test.go

```diff
@@ -1293,7 +1293,7 @@ func TestCacheWorkloadOperations(t *testing.T) {
 			}
 
 			gotError := step.operation(cache)
-			if diff := cmp.Diff(step.wantError, messageOrEmpty(gotError)); diff != "" {
+			if diff := cmp.Diff(step.wantError, utiltesting.MessageOrEmpty(gotError)); diff != "" {
```

Comment:
This escaped my review lens in the past :monocle_face: 

I'm not really a fan of checking errors by comparing strings. We should do the effort of implementing `Is` or exporting the relevant errors in the packages, so that we `cmp.Diff` can handle the rest.

But it can be left for a follow up.

Reply:
I see. I will work on cache pkg at follow-up PRs.